### ISSUE TITLE
[HOLD] Profile details not displayed clicking on the user in the header

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#20567]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. go to offline
2. click on + icon and select new chat option
3. search for email id which is not exist in list
4. click on that email
5. click on header to see details
## Expected Result: 
should show profile details
## Actual Result: 
shows list of profiles instead of profile details
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.26-1
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/8bdede6d-9a32-487d-a107-85c12006f7e0

https://github.com/Expensify/App/assets/43996225/79aa02c7-2f98-4afc-a1b4-5059de4b330f

**Expensify/Expensify Issue URL:**
**Issue reported by:** ~@gadhiyamanan~ @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1686300608119649
 
[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by johnmlee101** [comment]
Discussed with @Beamanator , we're not going to mark this as a blocker since any solution requires us to improve the keying system we use for optimistic accountIDs. 
https://github.com/Expensify/App/issues/19007

Going to remove the blocker, mark as held on this until that is complete.

**Comment by johnmlee101** [comment]
No update on this, since its still on HOLD

**Comment by bfitzexpensify** [comment]
https://github.com/Expensify/App/issues/19007 is live. I can no longer reproduce: 


https://github.com/Expensify/App/assets/38236554/b87578b5-fa69-421f-8d4c-ad23056ed4cf

@dhanashree-sawant @kavimuru can you reproduce?

**Comment by dhanashree-sawant** [comment]
Yes agree, cannot reproduce now

**Comment by bfitzexpensify** [comment]
Well, we were waiting on https://github.com/Expensify/App/issues/19007, and since that was deployed, this issue no longer exists. So I think we can just close this entirely?

**Comment by johnmlee101** [comment]
Agreed! Sounds fine to close
